structure_tests/structure_test_v1.go

```diff
@@ -195,7 +195,18 @@ func ProcessCommand(t *testing.T, envVars []EnvVar, fullCommand []string,
 		} else {
 			t.Fatalf("Error running setup/teardown command: %s.", err)
 		}
-		exitCode = err.(*exec.ExitError).Sys().(syscall.WaitStatus).ExitStatus()
+		exitErr, ok := err.(*exec.ExitError)
```

Comment:
Hmm, this might be cleaner with a type switch instead of the nested if statement/casting. WDYT?